src/main/java/cyclops/stream/ReactiveSeq.java

```diff
@@ -3875,19 +3875,17 @@ default T single(final Predicate<? super T> predicate) {
      */
     default <R> ReactiveSeq<R> retry(final Function<? super T, ? extends R> fn, final int retries, final long delay, final TimeUnit timeUnit) {
```

Comment:
@jijisv Could we move this impl up into Transformable? Then it will be available to all types that have retry (& map funcitonality in cyclops-react). Transformable in ReactiveSeq should ideally just call Transformable.super.retry (    ) . (If the super call works, I presume it does and the separate implementations is an oversight in the current code base, but there may also be a reason for the two impls).